airflow/contrib/operators/sftp_operator.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+from airflow.contrib.hooks.ssh_hook import SSHHook
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.utils.decorators import apply_defaults
+
+
+class SFTPOperator(BaseOperator):
+    """
+    SFTPOperator for transferring files from remote host to local or vice a versa.
+    This operator uses ssh_hook to open sftp trasport channel that serve as basis
+    for file transfer.
+
+    :param ssh_hook: predefined ssh_hook to use for remote execution
+    :type ssh_hook: :class:`SSHHook`
+    :param ssh_conn_id: connection id from airflow Connections
+    :type ssh_conn_id: str
+    :param remote_host: remote host to connect
+    :type remote_host: str
+    :param local_filepath: local file path to get or put
+    :type local_filepath: str
+    :param remote_filepath: remote file path to get or put
+    :type remote_filepath: str
+    :param operation: specify operation 'get' or 'put', defaults to get
+    :type get: bool
+    """
+
```

Comment:
As you're adding new functionality, could you make `template_fields` out of `local_filepath` and `remote_filepath`? I.e. add before `__init__` add:

```
template_fields = ('local_filepath', 'remote_filepath')
```

That's how I'm currently using your operator. :)